specs/validate_drp/LPM-17-PA2.yaml

```diff
@@ -0,0 +1,59 @@
+# PA2 specifications as defined in LPM-17, the Science Requirements
+# Document (SRD), http://ls.st/lpm-17
+# These specifications don't query against provenance; sub-specifications
+# *can* inherit these, though, to concretely test against a certain dataset.
+
+---
+# Specification partial
+id: 'PA2-base'
+threshold:
+  unit: 'mmag'
+  operator: '<='
+
+---
+# validate_drp.PA2_minimum_gri.srd
```

Comment:
Yeah, what do you think? I thought it'd be a nice practice to give the fully qualified name of a specification, especially when it's a highly inherited specification.